coderd/database/lock.go

```diff
@@ -13,7 +13,6 @@ const (
 	LockIDNotificationsReportGenerator
 	LockIDCryptoKeyRotation
 	LockIDReconcileTemplatePrebuilds
```

Comment:
I think we should use the name of the one we're deleting, rather.

Reply:
We're deleting: `DeterminePrebuildsState`, I guess it's not relevant anymore?
Because our lock is global lock which is held until reconciliation finish?

We can rename it to: `LockIDReconcileAllTemplatePrebuilds ` or `LockIDReconcileAllPrebuilds `?

Reply:
I agree

Reply:
done